.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile

```diff
@@ -141,18 +141,29 @@ pipeline {
                     steps {
                         unstash "source"
                         script {
-                            runWith(lib, failedTests, "eck-810-${BUILD_NUMBER}-e2e", "8.1.3")
+                            runWith(lib, failedTests, "eck-81-${BUILD_NUMBER}-e2e", "8.1.3")
                         }
                     }
                }
-               stage("8.2.2") {
+               stage("8.2.3") {
                     agent {
                         label 'linux'
                     }
                     steps {
                         unstash "source"
                         script {
-                            runWith(lib, failedTests, "eck-820-${BUILD_NUMBER}-e2e", "8.2.2")
+                            runWith(lib, failedTests, "eck-82-${BUILD_NUMBER}-e2e", "8.2.3")
+                        }
+                    }
+               }
+               stage("8.3.0") {
```

Comment:
Should we remove one of the older 7.x versions from testing to get the overall number of tests constant?